Cargo.toml

```diff
@@ -88,6 +88,8 @@ usb_hs = ["synopsys-usb-otg", "synopsys-usb-otg/hs"]
 
 sdio = ["sdio-host"]
 
+fmc = []
```

Comment:
Pleasy only add meta-features if absolutely required. The reason to have those for usb and sdio is because that's the only way to use optional external dependencies.